StockBugFix/StockBugsPatches.cs

```diff
@@ -581,5 +582,58 @@ public static class SpaceHeater_States_InitializeStates_Patch {
 					PUtil.LogWarning("No SpaceHeater update handler found");
 			}
 		}
+
+		/// <summary>
+		/// Applied to FoodDiagnostic.CheckEnoughFood to fix the calories calculation in the diagnostics panel.
+		/// </summary>
+		[HarmonyPatch(typeof(FoodDiagnostic), "CheckEnoughFood")]
+		public static class FoodDiagnostic_CheckEnoughFood_Patch {
```

Comment:
Since the diagnostic is not strictly broken in the base game, can you put this in `StockQOLPatches.cs` instead?

Reply:
Sure